src/todo_final.ts

```diff
@@ -0,0 +1,136 @@
+type TodoItem = {
+	checked: Boolean;
+	createdAt: Date; // Class Date as type, it will be the current date when the todo is created
+	dueAt: Date; //string to date
+	id: number;
+	text: string;
+};
+let todoItems: TodoItem[] = [];
+// this array will contain all the todo list items
```

Comment:
This comment is referring to `todoItems`, so it should be above `todoItems`, not below it.